### PR TITLE
Update azure-ad.html.md

### PR DESCRIPTION
**Pull request by abuxton**
malformed markdown due to missing line break between content and list.

## Labels

- [ ] inaccuracy
- [ ] clarification
- [ ] new docs
- [x] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

File: content/source/docs/cloud/users-teams-organizations/single-sign-on/azure-ad.html.md [hunk 1]
```diff
@@ -76,11 +76,13 @@ For more information on the listed features, visit the [Microsoft Azure AD SAML
 ## Team and Username Attributes
 
 To configure team management in your Microsoft Azure AD application:
+
 1. Navigate to the single sign-on page.
 1. Edit step 2, "User Attributes & Claims."
 1. We recomoned naming it "MemberOf", leaving the namespace blank, and potentially sourcing `user.assignedroles` as an easy starting point.
```
Comment (laurapacilio) [inline content/source/docs/cloud/users-teams-organizations/single-sign-on/azure-ad.html.md:82]:
```suggestion

1. Navigate to the single sign-on page.
2. Edit step 2, "User Attributes & Claims."
    We recommend naming it "MemberOf", leaving the namespace blank, and potentially sourcing `user.assignedroles` as an easy starting point.
```

File: content/source/docs/cloud/users-teams-organizations/single-sign-on/azure-ad.html.md [hunk 2]
```diff
@@ -76,11 +76,13 @@ For more information on the listed features, visit the [Microsoft Azure AD SAML
...
 
 If you plan to make use of SAML to set usernames in your Microsoft Azure AD application:
+
 1. Navigate to the single sign-on page.
 1. Edit step 2, "User Attributes & Claims"
 1. We recomend naming the claim "Username", leaving the namespace blank, and sourcing something like `user.displayname` or `user.mailnickname`.
```
Comment (laurapacilio) [inline content/source/docs/cloud/users-teams-organizations/single-sign-on/azure-ad.html.md:88]:
```suggestion
1. Navigate to the single sign-on page.
2. Edit step 2, "User Attributes & Claims."
    We recommend naming the claim "Username", leaving the namespace blank, and sourcing something like `user.displayname` or `user.mailnickname`.
```

**Review by laurapacilio — CHANGES_REQUESTED** [review]
Hi,

Thank you so much for submitting this fix! I'm suggesting a couple of additional changes that fix some other cosmetic issues with the text. Would you mind committing these as well? After you do that, I'll merge - thanks!

**Comment by abuxton** [comment]
@laurapacilio done thank you

**Review by laurapacilio — APPROVED** [review]

**Pull request merged**